portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,149 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+}
+
+body{
+   background-color: #f3f5f9;
+}
+
+.wrapper{
+  display: flex;
+  position: relative;
+}
+
+/* -------------- Side Bar --------------*/
+
+.wrapper .sidebar{
+  width: 280px;
+  height: 100%;
+  background: #6767b5;
+  padding: 30px 0px;
+  position: fixed;
+}
+
+/* -------------- Side Bar Avatar --------------*/
+
+.wrapper .sidebar .avatar {
+    position: relative;
+    overflow: hidden;
+    margin: 55px;
```

Comment:
55 here is a bit magic since if you ever change the width of the sidebar, this will need to be updated too. If you just want to center it, use a margin value of auto (for left/right).